src/components/Picklist/index.js

```diff
@@ -182,8 +207,18 @@ class Picklist extends Component {
     }
 
     openMenu() {
+        const { activeChildren } = this.state;
+        const firstOptionIndex = activeChildren.length > 0 ? 0 : -1;
+        const firstOptionName = activeChildren.length > 0 ? activeChildren[0].name : null;
+
+        setTimeout(() => {
+            this.scrollTo(0);
+        }, 0);
```

Comment:
let's try to avoid setTimeout, instead use didUpdate to check when open (!prevIsOpen && isOpen) the menu and then here is where you scroll without timeout